### ISSUE TITLE
Reorder questions should be doable without having to drag and drop

### ISSUE THREAD

**Issue by ghost** [xwiki-labs/cryptpad#1587]
Moving questions (for example duplicated ones) within a form should be possible without requiring to "drag and drop" the question.

Ideally, there would be two arrows somewhere on the interface, which would allow to change the position of the question, "up" or "down".
